src/V3Param.cpp

```diff
@@ -758,8 +758,24 @@ class ParamProcessor final {
         }
     }
 
+    void cellApplyDeparam(AstNode* nodep, AstNodeModule* paramedModp) {
```

Comment:
`cellDeparam` and `cellApplyDeparam` no longer only deal with cells. Let's get rid of the `cell` in their names. Also, `deparam` vs `applyDeparam` doesn't tell me much about the roles of these functions relative to each other. Can we find a more descriptive verb?